src/Package/Impl/Publishing/Handlers/RmdPublishHandler.cs

```diff
@@ -28,7 +29,7 @@ internal sealed class RmdPublishHandler : IMarkdownFlavorPublishHandler {
             string outputFile = Path.GetFileName(outputFilePath);
             string outputFolder = Path.GetDirectoryName(outputFilePath).Replace('\\', '/');
             // Run rmarkdown::render
-            return $"\"rmarkdown::render(\'{inputFile}\', output_format=\'{format}\', output_file='{outputFile}',  output_dir='{outputFolder}')\"";
```

Comment:
Well, there is a side effect since if file is modified, we do save it forcibly in UTF-8 which may not be what the user wants. Let me try and add the detection.